tests/pools/test_pool_rpc.py

```diff
@@ -448,7 +448,7 @@ def wallet_is_synced():
 
     @pytest.mark.asyncio
     @pytest.mark.parametrize("trusted", [True])
-    @pytest.mark.parametrize("fee", [0])
+    @pytest.mark.parametrize("fee", [0, FEE_AMOUNT])
```

Comment:
I'm not positive that we need the parameterization on the fee amount.  It seems like most fee logic lives under an `if fee > 0` statement which means we're only ever testing a superset of logic when we use a fee.  I don't care that much but it would cut down on test runtime if we were to remove this.

Reply:
I understand it's that way now, but if this code changes in the future, we want to test both paths.

Our test run-times do suck, but that's a separate issue.